jerry-main/main-unix-snapshot.c

```diff
@@ -436,6 +440,12 @@ static const cli_opt_t merge_opts[] =
 {
   CLI_OPT_DEF (.id = OPT_MERGE_HELP, .opt = "h", .longopt = "help",
                .help = "print this help and exit"),
+  CLI_OPT_DEF (.id = OPT_MERGE_LITERAL_LIST, .longopt = "save-literals-list-format",
+               .meta = "FILE",
+               .help = "export literals found in parsed JS input (in list format)"),
+  CLI_OPT_DEF (.id = OPT_MERGE_LITERAL_C, .longopt = "save-literals-c-format",
+               .meta = "FILE",
+               .help = "export literals found in parsed JS input (in C source format)"),
```

Comment:
Wouldn't it be better to have a separate subcommand for extracting literals from a snapshot instead of adding export options to both generate and merge subcommands? That would reduce code duplications also.

Reply:
I prefer it this way, but we can change it. Do you mean something like this:
`jerry-snapshot literal --format=list -o literal.list FILE_1 ... FILE_N` ?

Reply:
Yes, I was thinking something like this. It would have several benefits, IMO. 

The subcommands of the snapshot tool would work as analogies to well known tools from the native domain, e.g., 'generate' is like a compiler, 'merge' resembles a linker, and this 'literal' (or however it is named) could be the counterpart of objdump. (Ofc, none of them are perfect matches, there are important differences everywhere.)

There is a reason why these build tools are separated. This way a user doesn't have to execute unnecessary functionalities to perform a given step. E.g., in this case, assumig that a snapshot is already present but the user forgot to ask for a literal dump at the time of its creation, they don't have to recreate the snapshot from source just to get that dump but it can be retrieved from the snapshot itself.

All this is also aligned with the linux command line tool philosophy: a tool (now, subcommand) should do one thing but do that right. (I.e., don't clone the same functionality into different tools but put it into a single one and leave the functionality composition via sequential execution or pipes or whatever to the user.)